sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -185,6 +185,15 @@ trait CheckAnalysis extends PredicateHelper with LookupCatalog with QueryErrorsB
     )
   }
 
+  private def isContainsUnsupportedLCA(e: Expression, operator: LogicalPlan): Boolean = {
```

Comment:
Both `is` and `contains` are verbs.
The func should be named `containsUnsupportedLCA`

Reply:
+1

Reply:
Follow-up PR: https://github.com/apache/spark/pull/49134